tests/rptest/services/redpanda.py

```diff
@@ -3634,57 +3639,69 @@ def check():
-                try:
-                    status = self._admin.get_partition_cloud_storage_status(
-                        p.topic, p.index, node=p.leader)
-                except HTTPError as he:
-                    if he.response.status_code == 404:
-                        # Old redpanda, doesn't have this endpoint.  We can't
-                        # do our upload check.
-                        continue
-                    else:
-                        raise
+    def _ssh_output_stderr(self,
+                           node: ClusterNode,
+                           cmd,
+                           allow_fail=False,
+                           timeout_sec=None) -> tuple[bytes, bytes]:
+        """Runs the command via SSH and captures stdout and stderr, returning it as a byte strings.
+        this is a copy/mode of ssh_output, with the intention midterm to upstream it to ducktape
 
-                remote_write = status["cloud_storage_mode"] in {
-                    "full", "write_only"
-                }
-                has_uploaded_manifest = status[
-                    "metadata_update_pending"] is False or status.get(
-                        'ms_since_last_manifest_upload', None) is not None
-                if remote_write and not has_uploaded_manifest:
-                    self.logger.info(f"Partition {p} hasn't yet uploaded")
-                    return False
+        :param cmd: The remote ssh command.
+        :param node: where to run the command
+        :param allow_fail: If True, ignore nonzero exit status of the remote command,
+               else raise an ``RemoteCommandError``
+        :param timeout_sec: Set timeout on blocking reads/writes. Default None. For more details see
+            http://docs.paramiko.org/en/2.0/api/channel.html#paramiko.channel.Channel.settimeout
 
-            return True
+        :return: stdout, stderr pair output from the ssh command.
+        :raise RemoteCommandError: If ``allow_fail`` is False and the command returns a non-zero exit status
+        """
+        self.logger.debug(f"Running ssh command: {cmd}")
 
-        # If any nodes are up, then we expect to be able to talk to the cluster and
-        # check tiered storage status to wait for uploads to complete.
-        if self._started:
-            # Aggressive retry because almost always this should already be done
-            wait_until(all_partitions_uploaded_manifest,
-                       timeout_sec=30,
-                       backoff_sec=1)
+        client: SSHClient = node.account.ssh_client
+        stdin, stdout, stderr = client.exec_command(cmd, timeout=timeout_sec)
 
-        # We stop because the scrubbing routine would otherwise interpret
-        # ongoing uploads as inconsistency.  In future, we may replace this
-        # stop with a flush, when Redpanda gets an admin API for explicitly
-        # flushing data to remote storage.
-        self.stop()
+        try:
+            stdoutdata = stdout.read()
```

Comment:
This may not be a problem in practice, but FYI in case you hit issues:

Doing a `read()` on two streams one after the other may not always work: if there is data waiting to be read from stderr, then the read from stdout might get stuck.  This was a classic "gotcha" with `subprocess` & is why things like `subprocess.Popen.communicate` exist, although I'm not sure if there is an equivalent helper for paramiko.